test/testother.cpp

```diff
@@ -700,6 +701,20 @@ class TestOther : public TestFixture {
         ASSERT_EQUALS("[test.cpp:2:14]: (error) Division by zero. [zerodiv]\n", errout_str());
     }
 
+    void zeroDiv22()
+    {
+        check("int main() {\n"
+              "    int x = 20;\n"
```

Comment:
can the testcase be simplified.
 * variable x is not needed.
 * do we really need two loops, is one enough?
 * the "return 0;" does not have value in this test.